### PR TITLE
Categorize assets

### PR DESCRIPTION
**Pull request by yurishkuro**

### CODE REVIEW

File: assets.md [hunk 1]
```diff
@@ -2,29 +2,44 @@
 
 This file is intended to list all the assets controlled by OpenTelemetry.
 
-- Mailing list cncf-opentelemetry-net-maintainers@lists.cncf.io
-    - Used to register service accounts and as a NuGet OpenTelemetry organization e-mail. All .NET SIG maintainers are part of this list. Owned by CNCF.
-    - Admin: [add member](https://lists.cncf.io/g/cncf-opentelemetry-net-maintainers/).
+## CI pipelines
+
 - OpenTelemetry [Azure Pipeline](https://dev.azure.com/opentelemetry/).
```
Comment (yurishkuro) [inline assets.md:7]: @SergeyKanzhelev this is CI, right?

File: assets.md [hunk 2]
```diff
@@ -2,29 +2,44 @@
...
     - Owner: Mayur Kale @mayurkale22
     - Admin: Daniel Dyla @dyladan
     - Member: Bogdan Drutu @bogdandrutu
+
+## Communication channels
+
 - [Calendar Google Group](https://groups.google.com/forum/#!forum/opentelemetry-calendar)
     - Owner: Morgan McLean @mtwo
+
+- Mailing list cncf-opentelemetry-net-maintainers@lists.cncf.io
```
Comment (yurishkuro) [inline assets.md:33]: maybe this should be a bullet point under NuGet

File: assets.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply assets.md:7]: yes, but we switched to github actions. I can check and then remove

File: assets.md (hunk 2 above)
Comment (SergeyKanzhelev) [reply assets.md:33]: We also have one for ruby:

```
- Mailing list cncf-opentelemetry-ruby@lists.cncf.io
    - Used as a service account e-mail for Ruby SIG. All Ruby SIG maintainers [@open-telemetry/ruby-maintainers](https://github.com/orgs/open-telemetry/teams/ruby-maintainers) are moderators of this list. Owned by CNCF.
    - Moderators: [add member](https://lists.cncf.io/g/cncf-opentelemetry-ruby/members).
```

@open-telemetry/ruby-maintainers I see a new member on the maintainer team. @fbogsany @mwear I made you moderators so you should be able to add members. Or just contact me and let know who to add to the list

Comment (SergeyKanzhelev) [reply assets.md:33]: it's a service account. So maybe a separate category

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by arminru — APPROVED** [review]

File: assets.md [hunk 3]
```diff
@@ -2,29 +2,44 @@
...
 
-- Mailing list cncf-opentelemetry-net-maintainers@lists.cncf.io
-    - Used to register service accounts and as a NuGet OpenTelemetry organization e-mail. All .NET SIG maintainers are part of this list. Owned by CNCF.
-    - Admin: [add member](https://lists.cncf.io/g/cncf-opentelemetry-net-maintainers/).
+## CI pipelines
+
 - OpenTelemetry [Azure Pipeline](https://dev.azure.com/opentelemetry/).
     - Free tier is used.
     - Currently used for .NET SDK (.NET SIG maintainers are administrators).
     - Admin: [add administrator](https://dev.azure.com/opentelemetry/pipelines/_settings/)).
```
Comment (arminru) [inline assets.md:10]:
```suggestion
    - Admin: [add administrator](https://dev.azure.com/opentelemetry/pipelines/_settings/).
```

**Pull request merged**